src/compressor/QatAccel.h

```diff
@@ -37,9 +37,9 @@ class QatAccel {
 
   bool init(const std::string &alg);
 
-  int compress(const bufferlist &in, bufferlist &out, boost::optional<int32_t> &compressor_message);
-  int decompress(const bufferlist &in, bufferlist &out, boost::optional<int32_t> compressor_message);
-  int decompress(bufferlist::const_iterator &p, size_t compressed_len, bufferlist &dst, boost::optional<int32_t> compressor_message);
+  int compress(const bufferlist &in, bufferlist &out, std::optional<int32_t> &compressor_message);
+  int decompress(const bufferlist &in, bufferlist &out, std::optional<int32_t> compressor_message);
+  int decompress(bufferlist::const_iterator &p, size_t compressed_len, bufferlist &dst, std::optional<int32_t> compressor_message);
```

Comment:
probably worth updating all of [compressor/Compressor.h](https://github.com/ceph/ceph/blob/217c328/src/compressor/Compressor.h#L93-L97) while we're at it?

Reply:
sure. happy to do so.